src/typedefs.js

```diff
@@ -196,13 +196,26 @@ const React = require('react');
  * @memberof typedefs
  */
 
+/**
+ * documentTableProps is an object that stores the props for the DocumentTable
+ * component.
+ *
+ * @exports documentTableProps
+ * @typedef {object} documentTableProps
+ * @property {(documentName: string, modalType: string) => void} handleAclPermissionsModal
```

Comment:
I've just checked the function `handleAclPermissionsModal`. It seems like it takes in 3 parameters now with the parameter names being `modalType`, `docName`, and `docType`. This would need to be updated accordingly. I believe they're all still just strings? So we just need to change this to:

```@property {(modalType: string, docName: string, docType: string) => void}```

This should be true for Line 217 as well for `documentTableRowProps`.